internal/server/models/organization.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"github.com/infrahq/infra/api"
+	"github.com/infrahq/infra/uid"
+)
+
+type Organization struct {
+	Model
+
+	Name      string `gorm:"uniqueIndex:idx_organizations_name,where:deleted_at is NULL"`
+	CreatedBy uid.ID
+
+	Identities []Identity `gorm:"many2many:identities_organizations"`
+}
+
+func (o *Organization) ToAPI() *api.Organization {
+	return &api.Organization{
+		ID:      o.ID,
+		Created: api.Time(o.CreatedAt),
+		Updated: api.Time(o.UpdatedAt),
```

Comment:
might not want to expose created/updated for orgs

Reply:
This will only be available to the support role in the future, so I think it's probably OK.